ivy_tests/test_ivy/test_frontends/test_tensorflow/test_math.py

```diff
@@ -1044,7 +1044,7 @@ def test_tensorflow_truediv(
     dtype_and_x=statistical_dtype_values(function="argmin"),
     num_positional_args=helpers.num_positional_args(
         fn_name="ivy.functional.frontends.tensorflow.argmin"
-    ), output_types=_dtypes(),
+    ), output_types=helpers.get_dtypes("numeric"),
```

Comment:
This was fine as `output_types=_dtypes()`. The problem was you didn't copy and paste the `_dtypes` function to `test_math.py` so the function could not be found.